src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetPeerBlackListingResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void GetPeerBlackListingResponse_Can_Get_Output()
             //Arrange
             var setPeerBlackListRequest = new SetPeerBlacklistResponse();
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getPeerBlackListingCommand = new PeerBlackListingCommand(commandContext, Substitute.For<ILogger>());
+            new PeerBlackListingCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'PeerBlackListingCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)